manuscript.md

```diff
@@ -42,7 +46,35 @@
 
 ## How are interaction strengths actually inferred? 
 
-## Could we use hypergraphs and multi-layer networks to predict more interactions? 
+## Could we use hypergraphs and multi-layer networks to predict more interactions?
+
+The web of life is built by a variety of interactions that can vary over time
+and space; these interactions include facilitation, niche construction,
+zoonoses, vector-borne diseases, among others [@Garcia-Callejas2018MulInt],
+which all share the fact that interactions between species are themselves
+interacting. One mathematical tool to describe these situations is hypergraphs:
+hypergraphs are the generalization of a graph, allowing a broad and manageable
+approach to complex interactions [@Carletti2020DynSys]. @Golubski2011ModMod were
```

Comment:
I wonder id this one statement on what a hypergdaph is is sufficient. You are pointing the reader to a further reading opportunity so it probably is okay. Also after writing this and then checking the suggested glossary I see it is flagged there so I guess this is all good then - but maybe taking a step back in the glossary and expanding on the idea/concept of networks as graphs? Not sure if that comes up elsewhere in the ms....

Reply:
i think it needs to be more clear that a hypergraph considers edges between more than 2 nodes

Reply:
This is a good call, I will fix it in the next commit.